h263_codec_registration.src.html

```diff
@@ -0,0 +1,154 @@
+<pre class='metadata'>
+Title: H.263 WebCodecs Registration
+Repository: w3c/webcodecs
+Status: NOTE-ED
+Shortname: webcodecs-h263-codec-registration
+Level: none
+Group: mediawg
+ED: https://w3c.github.io/webcodecs/h263_codec_registration.html
+TR: https://www.w3.org/TR/webcodecs-h263-codec-registration/
+Editor: Yahweasel, Wereweasel https://wereweasel.com/
```

Comment:
@Yahweasel just to be clear, is it your intention to be on the hook for future maintenance (PR reviews) for this doc?

@tidoust, any issue with that? For my part, happy to have the help, but I don't know relevant W3C rules offhand.

Reply:
There is an issue in that, [per process](https://www.w3.org/2021/Process-20211102/#publication), "an editor must be a participant". I'm confident that we can find a suitable workaround though.

@Yahweasel could perhaps be granted an [invited expert status](https://www.w3.org/2021/Process-20211102/#invited-expert-wg), although note that has some licensing implications (the Patent Policy does not apply to this document, but the invited expert status is linked to the group as a whole, and the group develops normative specs), or simply switch to another official spec editor.

Reply:
I would warn you that I'm _not_, in particular, an expert. I guess I'm an expert in the way that anyone is an expert (I fell into the digital media field and learned while falling), but even as such, my expertise is far more in audio than video. This whole rabbit hole is scratching a particular itch for another project. I intend to maintain that project, of course, and to that end, this is part of that. I don't particularly mind being on the hook for this document, as I expect that (a) there's not much maintenance involved for a codification of a codec that hasn't changed in 15 years, and (b) no one else cares about H.263 :) .